lib/common_test/src/test_server_ctrl.erl

```diff
@@ -3616,28 +3620,32 @@ handle_test_case_io_and_status() ->
 			     (_) ->
 				  ok
 		  end, Cases),
-	    Result
+	    {{Ok, Skip, Fail}, Time}
     end.
 
 %% Handle cases (without Ref) that belong to the top parallel group (i.e. when Refs = [])
-handle_io_and_exit_loop([], [{undefined,CurrPid,CaseNum,Mod,Func}|Ps] = Cases, Ok,Skip,Fail) ->
+handle_io_and_exit_loop([], [{undefined,CurrPid,CaseNum,Mod,Func}|Ps] = Cases, Ok,Skip,Fail, Time) ->
     %% retrieve the start message for the current io session (= testcase)
     receive
 	{started,_,CurrPid,CaseNum,Mod,Func} ->
-	    {Ok1,Skip1,Fail1} =
+	    {Ok1,Skip1,Fail1,Time1} =
```

Comment:
Time, Time1, RTime - I would like those variable names to be more meaningful ... related to discussion about naming

CT source code is sometimes uneasy to read, let's try not to make it worse ...
maybe this also deserves a code comment section?